src/main/kotlin/app/revanced/cli/command/utility/InstallCommand.kt

```diff
@@ -44,20 +45,33 @@ internal object InstallCommand : Runnable {
                 }.install(Installer.Apk(apk, packageName))
             } catch (e: Exception) {
                 logger.severe(e.toString())
+                throw e
             }
 
             when (result) {
-                RootInstallerResult.FAILURE ->
+                RootInstallerResult.SUCCESS, AdbInstallerResult.Success ->
+                    logger.info("Installed the APK file")
```

Comment:
It seems like we already used "mount" in case RootInstallerResult.FAILURE. So in the success case for mount we can say it there too. What you had originally was fine, sorry!